src/dapp/walletConnectProvider.ts

```diff
@@ -132,6 +132,38 @@ export class WalletConnectProvider implements IDappProvider {
         return transaction;
     }
 
+    /**
+     * Sends a custom method and params and returns the response object
+     */    
+
+    async sendCustomMessage({
```

Comment:
Can you please add a link to this PR in the Changelog.md (the Unreleased section)? You can just follow the simplest example because we don't have breaking changes here to document

Reply:
done